tests/Integration/Database/EloquentModelTest.php

```diff
@@ -94,6 +94,35 @@ public function testDiscardChanges()
         $user->save();
         $this->assertFalse($user->wasChanged());
     }
+
+    public function testInsertRecordWithReservedWordFieldName()
+    {
+        Schema::create('actions', function (Blueprint $table) {
+            $table->id();
+            $table->string('label');
+            $table->timestamp('start');
+            $table->boolean('analyze');
+        });
```

Comment:
isn't the issue was with `end` column?

Reply:
I brought 'end' back in a commit but The `analyze` is also a keyword on PostgreSQL: https://www.postgresql.org/docs/current/sql-keywords-appendix.html
I removed 'end' because of unrelated MySQL 5.7 test failure when creating a table with 2 not null timestamp columns.